contracts/aggregator/polygon/flashloan/variables.sol

```diff
@@ -18,6 +19,9 @@ contract ConstantVariables {
     address public constant balancerLendingAddr = 0xBA12222222228d8Ba445958a75a0704d566BF2C8;
     IBalancerLending public constant balancerLending = IBalancerLending(balancerLendingAddr);
 
+    address public constant interopLendingAddr = address(0);
```

Comment:
```suggestion
    address public constant interopLendingAddr = address(0);
```
@Vaibhav20201 We need to update this, right?

Reply:
Yes, we do, will do it when we deploy the interop flashloan

Reply:
We have already deployed interopBeta contract. 

Since InteropBeta is upgradable contract. We just need to deploy the implementation and update the implementation on interopBeta.

Reply:
Updated!

Reply:
This is resolved?

Reply:
Had commited it. Forgot to push